Formula/docker.rb

```diff
@@ -2,8 +2,8 @@ class Docker < Formula
   desc "Pack, ship and run any application as a lightweight container"
   homepage "https://www.docker.com/"
   url "https://github.com/docker/docker.git",
-      :tag => "v17.05.0-ce",
-      :revision => "89658bed64c2a8fe05a978e5b87dbec409d57a0f"
+      :tag => "v17.06.0-ce",
```

Comment:
Right, the URL needs to be https://github.com/docker/docker-ce.git as he mentioned above